packages/clay-core/src/picker/Option.tsx

```diff
@@ -63,6 +63,11 @@ type Props = {
 	 */
 	index?: number;
 
+	/**
+	 * Flag to indicate that the option has href.
```

Comment:
When we say something is a flag it should be of type boolean. This comment should be identical to our other comments for hrefs in other components: https://github.com/search?q=repo%3Aliferay%2Fclay%20href%3F&type=code

Reply:
Thansk @ethib137 !